scassandra/src/main/scala/com/evolutiongaming/scassandra/QueryConfigImplicits.scala

```diff
@@ -0,0 +1,62 @@
+package com.evolutiongaming.scassandra
+
+import com.datastax.driver.core.ConsistencyLevel
+import com.evolutiongaming.scassandra.util.ConfigReaderFromEnum
+import pureconfig.ConfigReader
+
+import scala.concurrent.duration._
+
+trait QueryConfigImplicits {
+  implicit val configReaderConsistencyLevel: ConfigReader[ConsistencyLevel] = ConfigReaderFromEnum(ConsistencyLevel.values())
+
+  implicit val configReaderQueryConfig: ConfigReader[QueryConfig] =
```

Comment:
this looks very error-prone to me considering there are multiple parameters of the same type, it would be really easy to make a mistake and swap some names and end up with a wrong config...could we maybe parse it manually in the same fashion as `CassandraConfig` is parsed (see `CassandraConfig#fromConfig`)?

Reply:
I don't think we should use `ConfigSource` directly like in `CassandraConfig#fromConfig`. It's better to still expose `ConfigReaders` and preserve compatibility.

Changed to `ConfigReader.fromCursor`.